test/fixtures/BootstrapCommand/extra/lerna.json

```diff
@@ -0,0 +1,8 @@
+{
+  "lerna": "__TEST_VERSION__",
+  "version": "1.0.0",
+  "packages": [
+    {"glob": "packages/*/package.json"},
+    {"glob": "package-3/package.json"}
```

Comment:
I just submitted https://github.com/lerna/lerna/pull/393 which changes `packages` to an array of globs and does not require `package.json` to be specified, since I think that can be assumed. I also added a basic explanation to the README.